app/models/early_career_payments/slug_sequence.rb

```diff
@@ -26,6 +26,7 @@ class SlugSequence
       # personal details phase of claim journey
       "how_we_will_use_information_provided",
       "personal-details",
+      "gender",
```

Comment:
I find our use of `gender` and `payroll gender` confusing. Is this the name of the slug for the URL that our UCD colleagues gave us? Or an existing one for the current process in TSLR / M+P?

Reply:
The slug in the M&P journey is just labeled as "gender" - I went with sticking to the consistency of the codebase

Reply:
ok lets be pragmatic and we can alter once we have confirmation from UT / AC feedback if indeed it does need to change. I'm all in favour of consistency